community/iproute2/build

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+patch -p1 < 0001-make-iproute2-fhs-compliant.patch
+
+sed -i 's/-Werror//' Makefile
+
+./configure
```

Comment:
Missing `--prefix`? Can't `--sbindir` also be set here?

Reply:
I just copied arch on that, figured it's part of their base so it'd be a quality pkgbuild ...

Reply:
Gross: https://github.com/shemminger/iproute2/blob/master/configure

Reply:
ok well it works either way